.gitignore

```diff
@@ -14,3 +14,5 @@ __pycache__
 /graveyard/settings/production.py
 
 .DS_store
+
+docker-compose.yml
```

Comment:
`/docker-compose.yml` — compose in other directories should not be hidden by default